### PR TITLE
Return of Var on Cyberiad

### PR DESCRIPTION
**Pull request by Ninetoes3**
## What Does This PR Do
 
- Removes the forsaken cat from cargo maintenance and replaces it with the cat Var.

## Why It's Good For The Game

- Seen new players dying to the forsaken cat. 
- Gives player a new station pet.

## Testing

Succesfull compiled. Joined the game, played with the cat, fed cat mice, no runtime errors.

## Changelog

:cl:

del: Removed forsaken cat
add: Added the cat Var

/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 1]
```diff
@@ -176,6 +176,16 @@
...
 
+/mob/living/simple_animal/pet/cat/Var
+	name = "Var"
+	desc = "Maintenance Cat!"
+	icon_state = "cat2"
+	icon_living = "cat2"
+	icon_dead = "cat2_dead"
+	icon_resting = "cat2_rest"
+	gender = MALE
+	gold_core_spawnable = NO_SPAWN
```
Comment (Contrabang) [inline code/modules/mob/living/simple_animal/friendly/cat.dm:187]: All of these things are provided by inheritence from the normal cat subtype, so you can remove them here
```suggestion
/mob/living/simple_animal/pet/cat/Var
	name = "Var"
	desc = "Maintenance Cat!"
	gold_core_spawnable = NO_SPAWN
```

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 2]
```diff
@@ -75545,7 +75545,7 @@
 /turf/simulated/floor/plasteel,
 /area/station/service/janitor)
 "mSV" = (
-/mob/living/simple_animal/hostile/feral_cat/forsaken,
```
Comment (Contrabang) [inline _maps/map_files/cyberiad/cyberiad.dmm:75548]: /mob/living/simple_animal/hostile/feral_cat/forsaken isnt used anywhere else in the game, can you remove it please?

**Review by Contrabang — CHANGES_REQUESTED** [review]
I missed var so much

File: _maps/map_files/cyberiad/cyberiad.dmm (hunk 2 above)
Comment (Ninetoes3) [reply _maps/map_files/cyberiad/cyberiad.dmm:75548]: Removed the entry

**Review by Contrabang — APPROVED** [review]
LGTM!

**Review by DGamerL — APPROVED** [review]
Now all we need is for Runtime to be RnD's pet, and Darwin the CMO's pet!

**Pull request merged**